src/PDFPreviewer.tsx

```diff
@@ -235,9 +251,13 @@ function PDFPreviewer({
                 >
                     {pageViewports.length > 0 && (
                         <List
-                            style={{...styles.list, ...contentContainerStyle}}
+                            style={{
+                                ...styles.list,
+                                maxWidth: '100%', 
+                                ...contentContainerStyle
+                            }}
                             outerRef={setListAttributes}
-                            width={isSmallScreen ? pageWidth : containerWidth}
+                            width={calculatePageWidth()}
```

Comment:
```suggestion
                            width={pageWidth}
```
I think we can just use [pageWidth](https://github.com/Expensify/react-fast-pdf/blob/d47e034a046ac2bf2988d355932a1a9598d8e0e6/src/PDFPreviewer.tsx#L127C5-L127C44) value